addon/enums/header-buttons.js

```diff
@@ -0,0 +1,17 @@
+import Type from './types';
+
+export default Type.create({
+    HELP: {
```

Comment:
translations can either be added here or this enum can be overridden on a per-platform-basis - these stand as just sensible defaults